sources/api/migration/migrator/Cargo.toml

```diff
@@ -9,7 +9,11 @@ build = "build.rs"
 # Don't rebuild crate just because of changes to README.
 exclude = ["README.md"]
 
+[features]
+fips = ["tough/fips"]
```

Comment:
Does this need to be:
```
fips = ["tough/fips", "aws-lc-rs/fips"]
```

Reply:
Nope, `tough/fips` enables `aws-lc-rs/fips` feature flag.